tests/Unit/Service/RequestExceptionsTest.php

```diff
@@ -99,29 +100,39 @@ public function serviceUnavailableThrows(): void
 
     private function getService(Response $expectedResponse): RequestInterface
     {
-        $client = $this->createMock(Client::class);
-        $client->expects($this->once())
-            ->method('request')
+        $guzzleClient = $this->createMock(\GuzzleHttp\Client::class);
```

Comment:
why don't test GuzzleClient like in here: https://github.com/answear/wide-eyes-bundle/blob/master/tests/Unit/Service/AbstractClientTest.php ?

Reply:
I'll look at it later, thanks